src/sqlfluff/core/templaters/python.py

```diff
@@ -949,9 +952,7 @@ def _split_uniques_coalesce_rest(
 
             if starts[1] < stops[1] and last_owu_idx is not None:
                 # Yield the end bit
-                templater_logger.debug(
-                    "        Attempting Subsplit [post]: %s", sub_section
-                )
+                templater_logger.debug("        Attempting Subsplit [post].")
```

Comment:
I think this was just lazy code on my part - `sub_section` wasn't used in anything other than logging, so I've just removed it. 🤷